<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>difficultyWindow</class>
 <widget class="QDialog" name="difficultyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>385</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>485</width>
    <height>385</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>485</width>
    <height>385</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Battleship++</string>
  </property>
  <property name="windowIcon">
   <iconset resource="bsresource.qrc">
    <normaloff>:/logo/Sprites/logo.png</normaloff>:/logo/Sprites/logo.png</iconset>
  </property>
  <widget class="QPushButton" name="easyDiffButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:green;
font-size: 20px;</string>
   </property>
   <property name="text">
    <string>Easy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mediumDifficultyButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20px;</string>
   </property>
   <property name="text">
    <string>Medium</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hardDifficultyButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;
font-size: 20px;

</string>
   </property>
   <property name="text">
    <string>Hard</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt;</string>
   </property>
   <property name="text">
    <string>Select your Difficulty</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="bsresource.qrc"/>
 </resources>
 <connections/>
</ui>
